tests/test_libphsh_fortran_wrapper.py

```
@@ -126,10 +121,7 @@ def test_import_libphsh():
     try:
         from phaseshifts.lib import libphsh as _libphsh  # noqa: F401
     except ModuleNotFoundError:
-        assert False, (
-            "libphsh*{} has not been compiled.\n"
-            "Extension location search result: {}".format(ext, location)
-        )
+        assert False, "libphsh*{} has not been compiled.\n" "Extension location search result: {}".format(ext, location)
```

Ruff (0.14.10)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)